### ISSUE TITLE
Fullscreen video in Firefox

### ISSUE THREAD

**Issue by imjasonmiller** [jonhoo/configs#3]
Hi Jon!

I just made the switch to Arch due to your video and have been setting things up over the past week. I noticed a small issue with Firefox where going fullscreen still leaves the `Polybar` visible. 

I wondered whether you ran into this issue as well and, if so, if you managed to solve it. It seems a [fair amount of people](https://duckduckgo.com/?q=firefox+video+xmonad+fullscreen ) mention this issue, yet none of the solutions seem to have worked for me. Perhaps I glossed over something that you are aware of.

Here is my [`xmonad.hs`](https://github.com/imjasonmiller/configs/blob/0003497856704d1996d8ddef8cc71494a569c44e/gui/.xmonad/xmonad.hs) config.

Thanks for sharing the configs and many tips!

**Comment by jonhoo** [comment]
Hi!

Yeah, sadly this is an issue I've run into as well :'( I don't have a good solution for you. Fullscreen seems to work fine in other browsers, and seemingly _sometimes_ in Firefox. Fullscreen also works fine in other programs like `mpv`. If you make any progress, please do let me know!

**Comment by imjasonmiller** [comment]
Ha, at least I am not alone. I will make sure to update the issue if I come across a solution. Thanks for the reply, have a good day!

**Comment by hmble** [comment]
Maybe [DistroTube](https://youtu.be/qQ4CcMTLW-4) youtube video will help You.

Tl;dr `XMonad.Layout.NoBorders` docs will help you;)

**Comment by imjasonmiller** [comment]
@hmble is my understanding correct in that this will require shortcuts in order for it to go fullscreen? It seems @jonhoo has `XMonad.Layout.NoBorders` in [his configuration](https://github.com/jonhoo/configs/blob/2c4625a765e70e5fe92e2b16497ce2e3b91d55a2/gui/.xmonad/xmonad.hs#L154).

The issue in my case is that Firefox, unlike Google Chrome and VLC, fails to properly go fullscreen while  e.g. watching a video. 

Is this not the case for you?

**Comment by hmble** [comment]
I didn't cross checked jonhoo's config files. I am a beginner to WM. For now I am learning about bspwm
